ios/ble/peripheral/Characteristics.swift

```diff
@@ -5,72 +5,58 @@ typealias CharacteristicTuple = (properties: CBCharacteristicProperties, permiss
     case REQUEST_SIZE_CHAR_UUID = "00002031-0000-1000-8000-00805f9b34fb"
     case REQUEST_CHAR_UUID = "00002032-0000-1000-8000-00805f9b34fb"
     case RESPONSE_SIZE_CHAR_UUID = "00002033-0000-1000-8000-00805f9b34fb"
-    case RESPONSE_CHAR_UUID = "00002034-0000-1000-8000-00805f9b34fb"
-    case SEMAPHORE_CHAR_UUID = "00002035-0000-1000-8000-00805f9b34fb"
-    case VERIFICATION_STATUS_CHAR_UUID = "00002036-0000-1000-8000-00805f9b34fb"
-    case CONNECTION_STATUS_CHANGE_CHAR_UUID = "00002037-0000-1000-8000-00805f9b34fb"
+    case SUBMIT_RESPONSE_CHAR_UUID = "00002034-0000-1000-8000-00805f9b34fb"
+    case TRANSFER_REPORT_REQUEST_CHAR_UUID = "00002035-0000-1000-8000-00805f9b34fb"
+    case TRANSFER_REPORT_RESPONSE_CHAR_UUID = "00002036-0000-1000-8000-00805f9b34fb"
+    case VERIFICATION_STATUS_CHAR_UUID = "00002037-0000-1000-8000-00805f9b34fb"
+    case DISCONNECT_CHAR_UUID = "00002038-0000-1000-8000-00805f9b34fb"
 }
 
 let characteristicsMap: [String: CharacteristicTuple] = [
-    "00002030-0000-1000-8000-00805f9b34fb": (properties: CBCharacteristicProperties([.writeWithoutResponse, .write,]), permissions: CBAttributePermissions([.writeable]), value: nil),
+    "00002030-0000-1000-8000-00805f9b34fb": (properties: CBCharacteristicProperties([.writeWithoutResponse, .write]), permissions: CBAttributePermissions([.writeable]), value: nil),
     "00002031-0000-1000-8000-00805f9b34fb": (properties: CBCharacteristicProperties([.read, .indicate]), permissions: CBAttributePermissions([.readable]), value: nil),
     "00002032-0000-1000-8000-00805f9b34fb": (properties: CBCharacteristicProperties([.read, .indicate]), permissions: CBAttributePermissions([.readable]), value: nil),
     "00002033-0000-1000-8000-00805f9b34fb": (properties: CBCharacteristicProperties([.writeWithoutResponse, .write]), permissions: CBAttributePermissions([.writeable]), value: nil),
     "00002034-0000-1000-8000-00805f9b34fb": (properties: CBCharacteristicProperties([.writeWithoutResponse, .write]), permissions: CBAttributePermissions([.writeable]), value: nil),
-    "00002035-0000-1000-8000-00805f9b34fb": (properties: CBCharacteristicProperties([.writeWithoutResponse, .write]), permissions: CBAttributePermissions([.readable, .writeable]), value: nil),
-    "00002036-0000-1000-8000-00805f9b34fb": (properties: CBCharacteristicProperties([.read, .indicate]), permissions: CBAttributePermissions([.readable]), value: nil),
+    "00002035-0000-1000-8000-00805f9b34fb": (properties: CBCharacteristicProperties([.writeWithoutResponse]), permissions: CBAttributePermissions([.writeable]), value: nil),
+    "00002036-0000-1000-8000-00805f9b34fb": (properties: CBCharacteristicProperties([.indicate]), permissions: CBAttributePermissions([.readable]), value: nil),
+    "00002037-0000-1000-8000-00805f9b34fb": (properties: CBCharacteristicProperties([.indicate]), permissions: CBAttributePermissions([.readable]), value: nil),
```

Comment:
Is it intentional to not have disconnect char properties here?

Reply:
Before while i was refactoring uuid's on ios side the disconnect functionality is not implemented krishna and so they haven't added that uuid in this variable and thaanu is working on this functionality right now and this task is about renaming uuid's so i didn't touch it in this task

Reply:
Makes sense 👍